lib/zendesk_apps_support/assets/src.js.erb

```diff
@@ -1,32 +1,35 @@
+<% unless iframe_only %>
```

Comment:
I think roman and I thought there'd be too much "baggage" in the word "legacy", especially if we change again or add new features, so a feature test made more sense.